apps/web/playwright/organization/lib/inviteUser.ts

```diff
@@ -0,0 +1,57 @@
+import type { Page } from "@playwright/test";
+import type { createUsersFixture } from "playwright/fixtures/users";
+
+export const inviteUserToOrganization = async ({
+  page,
+  organizationId,
+  email,
+  usersFixture,
+}: {
+  page: Page;
+  organizationId: number;
+  email: string;
+  usersFixture: ReturnType<typeof createUsersFixture>;
+}) => {
+  await page.goto("/settings/organizations/members");
+  await page.waitForLoadState("networkidle");
+  const invitedUserEmail = usersFixture.trackEmail({
+    username: email.split("@")[0],
+    domain: email.split("@")[1],
+  });
+  await inviteAnEmail(page, invitedUserEmail);
+  return { invitedUserEmail };
+};
+
+export const inviteExistingUserToOrganization = async ({
+  page,
+  organizationId,
+  user,
+  usersFixture,
+}: {
+  page: Page;
+  organizationId: number;
+  user: {
+    email: string;
+  };
+  usersFixture: ReturnType<typeof createUsersFixture>;
+}) => {
+  await page.goto("/settings/organizations/members");
+  await page.waitForLoadState("networkidle");
+
+  await inviteAnEmail(page, user.email);
+  await page.waitForSelector('[data-testid="toast-success"]');
+  return { invitedUserEmail: user.email };
+};
+
+export async function acceptTeamOrOrgInvite(page: Page) {
+  await page.goto("/settings/teams");
+  await page.click('[data-testid^="accept-invitation"]');
+  await page.waitForLoadState("networkidle");
+}
+
+async function inviteAnEmail(page: Page, invitedUserEmail: string) {
```

Comment:
Same function here we could reuse: https://github.com/calcom/cal.com/pull/13576/files#diff-12fd7659258ab3763827234925e92c6aae8684b10d117e9bf372478586dec626R417

We already have data-testid defined for these buttons, let's use them